src/components/EndGame.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+
+class EndGame extends Component {
+
+    isUserGuessTheWord() {
```

Comment:
The name of the function refers that it returns a boolean. but, it actually returns an element